ush/global_ens/evs_global_ens_headline_plot.sh

```diff
@@ -126,7 +153,7 @@ for model in gfs gefs naefs ; do
     export output_base=$DATA/plot
     export stat_file_dir=$stat_data/$model
 
-   # stat_analysis -lookin $stat_data/$model/${model}_headline_*${yyyy}*.stat -fcst_valid_hour 00  -job aggregate_stat -line_type SAL1L2 -out_line_type CNT -by FCST_VAR,FCST_LEV,FCST_LEAD,VX_MASK -out_stat agg_stat_SAL1L2_to_CNT.${model}.${yyyy}.00Z
+    stat_analysis -lookin $stat_data/$model/${model}_headline_v*.stat -fcst_valid_hour 00  -job aggregate_stat -line_type SAL1L2 -out_line_type CNT -by FCST_VAR,FCST_LEV,FCST_LEAD,VX_MASK -out_stat agg_stat_SAL1L2_to_CNT.${model}.${yyyy}.00Z
 
    ${METPLUS_PATH}/ush/run_metplus.py -c ${PARMevs}/metplus_config/machine.conf -c ${PLOT_CONF}/StatAnlysis_fcstGENS_obsAnalysis_GatherByYear.conf 
```

Comment:
@BinbinZhou-NOAA I think that this is causing an error in the execution of the code.  See the .o file: /lfs/h2/emc/vpppg/noscrub/emc.vpppg/pr415test/EVS/dev/drivers/scripts/plots/global_ens/jevs_global_ens_headline_gefs_grid2grid_plots.o125961829

```
WARNING: 
WARNING: get_filenames() -> can't stat "/lfs/h2/emc/stmp/emc.vpppg/evs_test/prod/tmp/jevs_global_ens_headline_gefs_grid2grid_plots.125961829
.dbqs01/all_stats/gfs/gfs_headline_v*.stat"
WARNING: 
ERROR  : 
ERROR  : process_search_dirs() -> no STAT files found in the directories specified!
ERROR  : 
ERROR  : 
ERROR  : main() -> encountered an error value of 1.  Calling clean_up() and usage() before exiting.
ERROR  :
```

The stat_analysis command should be looking for ${model}_headline_**grid2grid**_v*.stat.  You're missing the grid2grid part of the file name.

Reply:
I am testing this with this code being ${model}_${VERIF_CASE}_headline.v*.stat.